server/service/transport.go

```diff
@@ -59,28 +59,41 @@ type htmlPage interface {
 	error() error
 }
 
-func idFromRequest(r *http.Request, name string) (uint, error) {
+func uintFromRequest(r *http.Request, name string) (uint64, error) {
```

Comment:
Pretty much all places where we convert to `uint` right now are places that will go away after migrating all endpoints - it will be done by the generic decoder of request structs, and will need to be `uint64` for `field.SetUint`.

Reply:
Gotcha!